myscript.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function(){
+    $("#first-name").html("Tony");
+    $("#last-name").html("Mainelli");
+    $(".mcc").css("color", "blue");
+    $(".mcc").css("font-style", "italic");
+    $("#hobby").append("<li>Watching Sci-fi movies and shows</li>")
+    $("#hobby").append("<li>Reading</li>")
```

Comment:
great job of using jQuery to target elements by ID and then alter or append! Equally nice job controlling the mcc class style! This is just what I would hope to see!